src/Module.Server/Rewards/CrpgRewardServer.cs

```diff
@@ -184,7 +199,7 @@ public override void OnRemoveBehavior()
                 userUpdate.BrokenItems = BreakItems(crpgPeer, durationRewarded * (lowPopulationServer ? 0.2f : 1f));
             }
 
-            userUpdates.Add(userUpdate);
+            userUpdates[crpgPeer] = userUpdate;
         }
 
         _ratingResults.Clear();

```

Comment:
Do not change code you do not need to change

Reply:
Done